views/js/controller/Delivery/monitoring.js

```diff
@@ -554,6 +556,10 @@ define([
                             if (row.state.status === 'INPROGRESS') {
                                 result = status.label;
                             }
+                            if (result === 'Awaiting') {
+                                // highlight row
+                                result = highlightedStatusTpl({status: result});
+                            }
```

Comment:
I have the same comments as for https://github.com/oat-sa/extension-tao-proctoring/pull/229: although this solution may works, what about exploring the possibility to extend the core datatable component to allow adding a class to a row under specific case like in this example. It will indeed be better if the highlight is not dependent from the need to manually tweak the cell with views/js/templates/delivery/highlightedStatus.tpl then search and set the line class in l.608.